src/en/models-cmr-scene-1.md

```diff
@@ -27,65 +31,69 @@ servicing several remote applications.
 
 ```bash
 juju add-model cmr-model
-juju deploy mysql --constraints "mem=16G root-disk=1T"
+juju deploy mysql --constraints "cores=4 mem=16G root-disk=1T"
 ```
 
-The output to `juju status` should eventually look similar to:
+!!! Note:
```

Comment:
I don't think this note is very noteworthy

Reply:
OK, but I wanted to make clear that a special CMR model is not required.

Reply:
I'm not actually sure that is helped by showing people the steps to create a controller and a model. We could just state that we have started with a model called whatever, and leave it to the reader to set up whatever model/controller they want.

Reply:
I feel that due to the various contexts (what controller, what model) it is important to be explicit.

Reply:
I just think the note is distracting. If it is felt to be important to mention you can use any model, you could just mention that when you create the ones above. It isn't a big deal, it just feels like a big interruption for the reader for something minor(is)

Reply:
I removed the Note.